src/com/spikes2212/utils/CamerasHandler.java

```diff
@@ -0,0 +1,58 @@
+package com.spikes2212.utils;
+
+import java.util.ArrayList;
+import java.util.function.Supplier;
+
+import org.opencv.core.Mat;
+import org.opencv.ml.SVM;
+
+import com.spikes2212.dashboard.ConstantHandler;
+
+import edu.wpi.cscore.CvSink;
+import edu.wpi.cscore.CvSource;
+import edu.wpi.cscore.UsbCamera;
+import edu.wpi.first.wpilibj.CameraServer;
+
+public class CamerasHandler {
+	private int port;
+	private ArrayList<UsbCamera> cameras = new ArrayList<>();
+	private ArrayList<CvSink> cvSinks = new ArrayList<>();
+	private ArrayList<Integer> ports = new ArrayList<>();

```

Comment:
Don't save the ports in a different object from the sinks please. Do a map, or put every sink in the index corresponding to its port?